src/shared/buttons/restaurants-button/restaurants-button.component.html

```diff
@@ -0,0 +1,4 @@
+<button class="container">
+  All Restaurants
+  <img src="../../../assets/right-arrow.svg" alt="link to restaurants" />
+</button>
```

Comment:
I would try to build a component for every button which route you to another page - which can get text and src for url
For example
<link-button
[text]="All Restaurant"
[imageUrl]= "right-arrow.svg"
>
If button have the same logic, I would think maybe to bind the specific route when click on the button and implement the route inside the button's component.
Speak with me if something is not understandable